### PR TITLE
adding country administration

### PR DESCRIPTION
**Pull request by sona-podholova**
# TL;DR

Ability to manage countries

# Proof
image

# Merge request checklist

Please check if your merge request fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (yarn build) was run locally and any changes were pushed
- [ ] API (yarn api:dev) runs locally and any fixes were made for failures
- [ ] Admin Client (cd clients/admin && yarn start) runs locally and any fixes were made for failures

# Merge request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Before this merge request
Prior to this change, it was not possible to edit or add countries or view their details

# How you fixed what was wrong
All pages needed to edit, add countries or view their details have been added, mostly as copies and pastes of product related pages

### CODE REVIEW

File: clients/admin/src/components/add-country/AddOrEditCountry.svelte [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        {
+            initCheck: true,
+            validateOnChange: false,
+            stopAtFirstError: false,
+            stopAtFirstFieldError: false,
+        },
+    );
+
+    export const saveCountryMutation = mutation<SaveCountryMutation, SaveCountryMutationVariables>(
+        SAVE_PRODUCT,
```
Comment (davidpodhola) [inline clients/admin/src/components/add-country/AddOrEditCountry.svelte:44]: please rename

File: clients/admin/src/lib/country.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import gql from 'graphql-tag';
+import type {
+    CountryByIdQuery,
+    CountryListPartsFragment,
+    CountriesQuery,
+} from '../generated/graphql';
+import { query } from 'svelte-apollo';
+import { store } from './store';
+import { COUNTRY_DETAIL_PARTS } from './fragments';
+import { PRODUCTS } from './queries/countries';
```
Comment (davidpodhola) [inline clients/admin/src/lib/country.ts:10]: please rename

File: clients/admin/src/pages/Countries.svelte [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+    import { query } from 'svelte-apollo';
+    import type { CountriesQuery } from 'src/generated/graphql';
+    import { apollo, setClient } from '../lib/apollo';
+    import { getError } from '../lib/util';
+    import CountryList from '../components/countries/CountryList.svelte';
+    import { urls } from './pathAndSegment';
+    import { PRODUCTS } from '../lib/queries/countries';
```
Comment (davidpodhola) [inline clients/admin/src/pages/Countries.svelte:8]: please rename

**Review by davidpodhola — CHANGES_REQUESTED** [review]

**Review by davidpodhola — APPROVED** [review]

**Comment by davidpodhola** [comment]
Great job!
